app/src/main/java/tech/ula/ui/FilesystemEditFragment.kt

```diff
@@ -81,6 +96,33 @@ class FilesystemEditFragment : Fragment() {
                 filesystem.distributionType = parent?.getItemAtPosition(position).toString()
             }
         }
+
+        restore_filesystem_button.setOnClickListener {
+            val intent = Intent(Intent.ACTION_GET_CONTENT)
```

Comment:
I don't like the reliance on random 3rd party apps for this flow. We need a way to track backups in external storage so that they can be retrieved internally.

Reply:
Ok. More in comments below..